components/base/checkbox.vue

```diff
@@ -2,6 +2,7 @@
 defineProps<{
   label: string
   description?: string
+  isDisabled?: boolean
```

Comment:
```suggestion
  disabled?: boolean
```
Osobiście przy tworzeniu wrapperów na HTMLowe komponenty stosuje domyślne nazewnictwo (np. disabled, type, readonly, ...)